tool/sel4coreplat/__main__.py

```diff
@@ -1460,6 +1462,10 @@ def build_system(
     for tcb_obj, schedcontext_obj, pd in zip(tcb_objects, schedcontext_objects, system.protection_domains):
         system_invocations.append(Sel4TcbSetSchedParams(tcb_obj.cap_addr, INIT_TCB_CAP_ADDRESS, pd.priority, pd.priority, schedcontext_obj.cap_addr, fault_ep_endpoint_object.cap_addr))
 
+    # Copy the PD's TCB cap into their address space for development purposes.
```

Comment:
I *think* that this is only needed for development purposes, and following POLA we'd ideally avoid having this cap available in production.

We probably need to think about the best way to support development cap needs vs production cap needs. Could be topic for a Monday meeting discussion. I'll think some more about it.